scripts/Muon/GUI/Common/muon_load_data.py

```diff
@@ -137,3 +137,17 @@ def get_main_field_direction(self, **kwargs):
             return self.get_data(**kwargs)['workspace']['MainFieldDirection']
         else:
             return None
+
+    def remove_workspace_by_name(self, workspace_name, instrument=''):
+        list_of_workspace_names_to_remove = []
+        for entry in self.params:
+            if any([workspace.workspace_name == workspace_name for workspace in entry['workspace']['OutputWorkspace']]):
+                list_of_workspace_names_to_remove.append(entry)
+
+        runs_removed = []
+        for entry in list_of_workspace_names_to_remove:
+            if instrument == entry['instrument']:
+                runs_removed.append(entry['run'])
+            self.remove_data(**entry)
```

Comment:
what does the ** mean?

Reply:
This unrolls the dictionary object entry so that it's values are passed as keyword arguments to the function